packages/react-core/src/next/components/Modal/ModalBoxBody.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/ModalBox/modal-box';
+
+export interface ModalBoxBodyProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside the modal box body. */
+  children?: React.ReactNode;
+  /** Additional classes added to the modal box body. */
+  className?: string;
+  /** Accessible label applied to the modal box body. This should be used to communicate
+   * important information about the modal box body div element if needed, such as that it
+   * is scrollable.
+   */
+  'aria-label'?: string;
+  /** Accessible role applied to the modal box body. This will default to "region" if the
+   * bodyAriaLabel property is passed in. Set to a more appropriate role as applicable
+   * based on the modal content and context.
+   */
+  'aria-role'?: string;
```

Comment:
We're using just `role` rather than `aria-role` in other components, also quick verbiage update

```suggestion
  /** Accessible role applied to the modal box body. This will default to "region" if the
   * aria-label property is passed in. Set to a more appropriate role as applicable
   * based on the modal content and context.
   */
  role?: string;
```